.github/ISSUE_TEMPLATE/01_bug_report.yml

```diff
@@ -19,7 +19,10 @@ body:
   - type: input
     attributes:
       label: "`semantic-release` version"
-      description: Version set in `package.json` `devDpendencies`.
+      description: |
+        The version executed in your pipeline, printed in the first line of output.
```

Comment:
we dont encourage defining semantic-release as a dependency, so looking in the log output is a better source to suggest finding this detail than `devDependencies`